modules/silauth/src/Auth/Source/yii

```diff
@@ -8,8 +8,8 @@
  * @license http://www.yiiframework.com/license/
  */
 
-define('YII_DEBUG', false);
-define('YII_ENV', 'production');
+const YII_DEBUG = false;
```

Comment:
The define function is better, because if it is ALREADY DEFINED, it doesn't change it.

Reply:
Well, PhpStorm didn't say anything about that. Thanks. But I don't think that's an issue here, since `yii` is always the first bit to run and these are at the top of the file. [This SO answer](https://stackoverflow.com/a/3193704/1461333) makes it sound like `const` is generally preferred.

Reply:
If you have a docker compose environment variable set, this would override that value. Define would not.

Reply:
These aren't environment variables, though.